### ISSUE TITLE
Docker daemon hosted in WSL 2: Docker fails to connect at open //./pipe/docker_wsl

### ISSUE THREAD

**Issue by sc0ttwad3** [docker/for-win#5105]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID:
	69CDC418-3206-4197-A18B-FB4AF7BE5977/20191109040906
### Expected behavior
Normal docker operations with new functionality to remain operable.

### Actual behavior
With the Docker daemon hosted in WSL 2: Docker fails to connect during open //./pipe/docker_wsl

### Information

  - Windows Version: Windows 10 Insider build 19018
  - Docker Desktop Version: 
		Docker Desktop Community 2.1.5.0
			Kubernetes 1.15.5
			Docker Compose 1.25.0-rc4, build 8f3c9c58
			Linux kernel 4.19.76
			docker-py version: 4.1.0
			CPython version: 3.7.4
			OpenSSL version: OpenSSL 1.1.1c  28 May 2019
  - Is it reproducible?
	YES
  - Is the problem new?
	YES
  - Did the problem appear with an update?
	YES, after the latest Windows 10 Insider update to build 19018.
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM:
NO

### Steps to reproduce the behavior

  0. ...Must be on a Windows 10 Insider build < 19018
  1. ...Update 2.1.4.0 to Docker Desktop for Windows Edge Release 2.1.5.0
  2. ...Verify settings are correct for WSL 2 engine (see last images below)
  3. ...Verify normal docker operations
  4. ...Update Windows 10 to build 19018
  5. ...Attempt normal docker operations

```
$ docker context ls
NAME                DESCRIPTION                               DOCKER ENDPOINT                  KUBERNETES ENDPOINT                                 ORCHESTRATOR
default             Current DOCKER_HOST based configuration   npipe:////./pipe/docker_engine   https://kubernetes.docker.internal:6443 (default)   kubernetes
wsl *               Docker daemon hosted in WSL 2             npipe:////./pipe/docker_wsl
```

```
$ docker images -a
error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_wsl/v1.40/images/json?all=1: open //./pipe/docker_wsl: The system cannot find the file specified.
```

```
$ docker version
Client: Docker Engine - Community
 Version:           19.03.4
 API version:       1.40
 Go version:        go1.12.10
 Git commit:        9013bf5
 Built:             Thu Oct 17 23:44:48 2019
 OS/Arch:           windows/amd64
 Experimental:      true
error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_wsl/v1.40/version: open //./pipe/docker_wsl: The system cannot find the file specified.
```

In Ubuntu-18.04 WSL 2 

```
$ pstree
init()─┬─init(35)───init(35)───docker-desktop-(37)─┬─{docker-desktop-}(37)
       │                                           ├─{docker-desktop-}(37)
       │                                           ├─{docker-desktop-}(37)
       │                                           ├─{docker-desktop-}(37)
       │                                           ├─{docker-desktop-}(37)
       │                                           ├─{docker-desktop-}(37)
       │                                           └─{docker-desktop-}(37)
       ├─init(136)───init(136)───zsh(138)───pstree(189)
       └─{init}()
```

![docker_wsl2_001](https://user-images.githubusercontent.com/1682691/68523285-ee967d80-0284-11ea-8c81-fe33d514444f.jpg)
![docker_wsl2_002](https://user-images.githubusercontent.com/1682691/68523289-f3f3c800-0284-11ea-8b62-5e1c49fca888.jpg)
![docker_wsl2_003](https://user-images.githubusercontent.com/1682691/68523291-f9511280-0284-11ea-827d-3b835cec4721.jpg)

**Comment by khs1994** [comment]
`$ docker context use default`

**Comment by aetos382** [comment]
I have the same problem.

**Comment by sc0ttwad3** [comment]
@khs1994 Thanks, that worked.

I was afraid of that, something obvious and right in my face. How the windows build update affected the docker context, I have no idea.

**Comment by dannyheardairelogic** [comment]
Still having this issue every time I reboot. Running windows insider version 19023 and docker desktop 2.1.50. There must be a better solution that having to reset the context every time?

**Comment by dannyheardairelogic** [comment]
Seems to be fixed in 2.1.6.0
